### DISCUSSION TITLE
RPI does not take into account the microphone

### DISCUSSION THREAD

**Discussion by Polyvalence** [mcguirepr89/BirdNET-Pi#720] [Q&A] [answered]
Hi. My Birdnet-pi records sounds, but it seems to be the RPI sound itself : it's like interference, or maybe the fan. I've plugged the microphone in each on the upper USB port (left and right), problem is still present. And I've tested the microphone on my computer : it works very nice (I get the devices from https://pixcams.com/building-a-birdnet-pi-real-time-acoustic-bird-id-station/).

I've also changed the "Audio card" : "default", "pulse", "lavrate", "samplerate", "oos" (everything available in "arecord -L" ;-) ). Not better.

What could be wrong in my installation ?
Thanks for your replies.

Pierre

![birdnet-luttre-BE — Mozilla Firefox](https://user-images.githubusercontent.com/16412800/215716951-3b4816f6-34de-4068-986f-8c6ce678acf9.jpg)

**Comment by FirstTenor** [comment]
There is a discussion on a very similar issue further down in the list ‘Problem setting sound card’.  It is worthwhile checking that. …. And particularly where there is a description of how to use that information from arecord -L ….. and the Journal to -f function to see what is happening.

**Comment by Polyvalence** [comment]
I've searched ahead.
After having changed the power supply, from POE (POE splitter) to standard USB-C, I no longer have any problem. I confirm noise was something coming from interference, ...
I will try to understand why (maybe another POE switch and/or splitter ?).

**Comment by Polyvalence** [comment, marked as answer]
So, it's fixed :-)
Problem was the POE switch. After lot of tests (direct ethernet connection in the wall, switch off all wave sources around the Raspberry), I've finally replaced the POE switch into a standard switch (same manufacturer, TP-Link), of course now my Raspberry is powered via USB-C. Recorded sound is better and Birdnet can detect a lot of birds.
Next job will be the improvement of the recorded sound.
